content/2018-09-09-newsletter-11.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "The Embedded Working Group Newsletter - 11"
+date = 2018-09-09
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This is the eleventh newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
+
+[Embedded WG]: https://github.com/rust-embedded/wg
+
+Discuss on [internals.rust-lang.org], [on twitter], or [on reddit]!
+
+<!-- TODO, update links after post lands -->
+[internals.rust-lang.org]: #
+[on twitter]: #
+[on reddit]: #
+
+<!-- more -->
+
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-embedded/blog/issues/12
+
+## Highlights
+
+* `rustc` now supports 4 **Cortex-R** targets thanks to the work of [paoloteti] in the Cortex-R space! You can build programs for these targets using nothing but the Rust toolchain
+* [James Munns] gave a talk at RustConf 2018, covering the basics of embedded systems, and how Rust's Zero Cost Abstractions are a perfect match for bare metal systems
```

Comment:
Should "at RustConf 2018" on this line be surrounded with square brackets?

Reply:
Yup, will fix in the next PR. Good catch!